examples/react-gallery/src/Store.js

```diff
@@ -0,0 +1,18 @@
+export const Store = (id) => {
```

Comment:
name 'Store' is a little confusing and I think we should abstract this away from user eventually anyway - will pass review for now though and added a new issue for this -

https://github.com/functionland/fula/issues/86

Reply:
i think i should remove it. i don't use it. it make a sample to complicated